templates/jmxtrans.service.epp

```diff
@@ -5,10 +5,18 @@ After=syslog.target network.target
 [Service]
 Type=forking
 User=<%= $user %>
+<% if $jmxtrans::systemd_environment_vars =~ Array[String] { -%>
+<% $jmxtrans::systemd_environment_vars.flatten.each |$var| { -%>
+Environment=<%= $var %>
+<% } -%>
+<% } -%>
+<% if $jmxtrans::working_directory { -%>
```

Comment:
This is fine (besides switching to a local parameter variable rather than a namespaced variable), although I'm curious what the use case is.